config/initializers/sdr.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# Configure dor-services-client to use the dor-services URL
+Dor::Services::Client.configure(url: Settings.dor_services.url,
+                                token: Settings.dor_services.token)
```

Comment:
I like the explicitness of this being an initializer, does this mean all calls through `Dor::Services::Client` have this configuration? I like that.

Reply:
Yes. Many (but not all) of our clients are so configured.